Taskfile.yml

```diff
@@ -158,3 +160,22 @@ tasks:
     desc: Accept all test snapshots. (Shortcut to accept snapshots after running tests)
     cmds:
       - Rscript -e 'testthat::snapshot_accept()'
+
+  format-r:
+    desc: Format R files.
+    aliases:
+      - fmt-r
+    sources:
+      - "{{.R_SOURCE}}"
+      - "{{.R_TESTS}}"
+      - air.toml
+      # These files are auto-generated and ignored by the formatter
+      - exclude: R/000-wrappers.R
+      - exclude: R/import-standalone-*.R
+    generates:
+      - "{{.R_SOURCE}}"
+      - "{{.R_TESTS}}"
+      - exclude: R/000-wrappers.R
+      - exclude: R/import-standalone-*.R
+    cmds:
+      - air format R/ tests/
```

Comment:
Can we put this in #73 instead?

I don't understand this PR, I thought it was just to set `air.toml`.

Reply:
I was checking the behavior here because these have interactions; it's OK to have them cherry-picked at #73 and removed here.